spec/api-menu-spec.js

```diff
@@ -283,7 +283,52 @@ describe('Menu module', () => {
     })
 
     it('returns immediately', () => {
-      menu.popup(w, {x: 100, y: 100, async: true})
+      const { browserWindow, x, y } = menu.popup(w, {x: 100, y: 101})
+
+      assert.equal(browserWindow, w)
+      assert.equal(x, 100)
+      assert.equal(y, 101)
+
+      menu.closePopup(w)
+    })
+
+    it('works without a given BrowserWindow and options', () => {
+      const { browserWindow, x, y } = menu.popup({x: 100, y: 101})
+
+      assert.equal(browserWindow.constructor.name, 'BrowserWindow')
+      assert.equal(x, 100)
+      assert.equal(y, 101)
+
+      menu.closePopup()
```

Comment:
Shouldn't the `browserWindow` be passed to the `menu.closePopup()` call?

Reply:
The parameter is optional, so if you opened it without one (which uses the focused window), it should just close. Then again, I'm not sure how well getting the key window on CI works, so we could pass it here.